Please rebase on top of the latest dev and fix this warning:
```
../librz/type/function.c: In function ‘rz_type_callable_free’:
../librz/type/function.c:58:15: error: passing argument 1 of ‘free’ discards ‘const’ qualifier from pointer target type [-Werror=discarded-qualifiers]
   58 |  free(callable->name);
      |       ~~~~~~~~^~~~~~
In file included from ../librz/include/rz_types.h:251,
                 from ../librz/include/rz_util.h:7,
                 from ../librz/type/function.c:7:
/usr/include/stdlib.h:565:25: note: expected ‘void *’ but argument is of type ‘const char *’
  565 | extern void free (void *__ptr) __THROW;
      |                   ~~~~~~^~~~~
```
Apart from that - good catch and thank you!

--- a/librz/include/rz_type.h
+++ b/librz/include/rz_type.h
@@ -143,7 +143,7 @@ typedef struct rz_callable_arg_t {
 } RzCallableArg;
 
 typedef struct rz_callable_at {
-	RZ_NULLABLE const char *name; // optional
+	RZ_NULLABLE char *name; // optional
 	RZ_NULLABLE RzType *ret; /// optional for the time being
 	RzPVector /* RzCallableArg */ *args;
 	RZ_NULLABLE const char *cc; // optional

--- a/librz/type/parser/types_parser.c
+++ b/librz/type/parser/types_parser.c
@@ -90,7 +90,7 @@ int parse_primitive_type(CParserState *state, TSNode node, const char *text, Par
 		node_malformed_error(state, node, text, "not primitive type");
 		return -1;
 	}
-	const char *real_type = ts_node_sub_string(node, text);
+	char *real_type = ts_node_sub_string(node, text);
 	if (!real_type) {
 		node_malformed_error(state, node, text, "primitive type");
 		parser_error(state, "Primitive type name cannot be NULL\n");

--- a/librz/type/serialize_functions.c
+++ b/librz/type/serialize_functions.c
@@ -144,6 +144,7 @@ static bool sdb_load_callables(RzTypeDB *typedb, Sdb *sdb) {
 		}
 	}
 	ht_pp_free(type_str_cache);
+	ls_free(l);
 	return true;
 }
 

--- a/librz/type/serialize_types.c
+++ b/librz/type/serialize_types.c
@@ -347,6 +347,7 @@ bool sdb_load_base_types(RzTypeDB *typedb, Sdb *sdb) {
 		}
 		free(tpair);
 	}
+	ls_free(l);
 	return true;
 }
 

